### PR TITLE
add technical posts in projects page

### PR DESCRIPTION
**Pull request by erizocosmico**
depends on #28 
review just last commit
screenshot_2017-06-15_11 41 42

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -67,6 +65,10 @@ function setupShowcases() {
   const showcase = $('.showcases__project')
   const code = document.getElementById('showcase-code')
   const title = document.getElementById('showcase-title')
+  if (!code) {
```
Comment (dpordomingo) [inline src/js/index.js]: you could move the return higher doing:
```
if (!showcase) return
```

File: hugo/data/projects.yml [hunk 2]
```diff
@@ -1,2 +1,8 @@
 title: projects
 desc: "source{d} is building the open-source components that enable matchine learning on source code. We believe that the next step in programming comes from teaching machines to understand and write code."
+posts:
+  display: true
+  categories:
+    - category: technical
```
Comment (dpordomingo) [inline hugo/data/projects.yml]: already defined at `data/home/home.yml`; the blog part could be moved to `data/blog.yml`

File: hugo/layouts/partials/projects/blog.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<section class="blog band band_grey" id="blog-container">
```
Comment (dpordomingo) [inline hugo/layouts/partials/projects/blog.html]: It could be reused the `partials/home/blog.html` once moved to `partials/blog.html`

Comment (erizocosmico) [inline hugo/layouts/partials/projects/blog.html]: Nope. This has the h1 in another place. Would need to be configured via more hugo `dicts` and rather die

Comment (Serabe) [inline hugo/layouts/partials/projects/blog.html]: Why does it need to be in a different place?

File: src/js/index.js (hunk 1 above)
Comment (Serabe) [inline src/js/index.js]: Nope. !showcase will always be false. OTOH, why jQuery for one of the selectors?

Comment (Serabe) [inline src/js/index.js]: nvm, saw the slick thing.

Comment (erizocosmico) [inline src/js/index.js]: Yep, the slick thing

File: hugo/layouts/partials/projects/blog.html (hunk 3 above)
Comment (erizocosmico) [inline hugo/layouts/partials/projects/blog.html]: Hmm, well, actually... I think I have an idea to reuse this.

**Comment by erizocosmico** [comment]
Done @Serabe @dpordomingo

File: src/js/index.js (hunk 1 above)
Comment (dpordomingo) [inline src/js/index.js]: @Serabe `$('.showcases__project')` returns `null` if there is no elements matching the selector

**Comment by erizocosmico** [comment]
Done @dpordomingo

**Pull request merged**